erpnext_shopify/sync_orders.py

```diff
@@ -34,7 +34,7 @@ def valid_customer_and_product(shopify_order):
 	customer_id = shopify_order.get("customer", {}).get("id")
 	if customer_id:
 		if not frappe.db.get_value("Customer", {"shopify_customer_id": customer_id}, "name"):
-			create_customer(shopify_order.get("customer"))
+			create_customer(shopify_order.get("customer"), shopify_customer_list=[])
```

Comment:
Won't this always create a customer in Shopify? as per line 106 of sync_customers.py?